src/Shop.java

```diff
@@ -0,0 +1,167 @@
+        } while (runMenu);
+    }
+
+    private void showTitle() {
+        System.out.println("Show All Bikes Enter : a");
+        System.out.println("Add New Bike Enter : n");
+        System.out.println("Exit Enter : e");
+        System.out.println("Change Bike Enter : c");
+        System.out.println("Delete Bike Enter : d");
+    }
+
+    private void showAllBikes() {
+        int bikeSize = bikes.size();
+        for (int bikeId = 0; bikeId < bikeSize; bikeId++) {
+            System.out.println("Id : " + bikeId);
+            showBike(bikeId);
+        }
+    }
+
+    private void showBike(int bikeId) {
+        System.out.println(bikes.get(bikeId).getTextBikePresentation());
+    }
+
+    private int choiceBike() {
+        System.out.println("Choice Bike Enter Id : ");
+        int choice = Integer.parseInt(UserInput.get());
+        return choice;
+    }
+
+    private void choiceValueBike(int idBike) {
+        System.out.println("Change : Title Enter t , Price Enter p , Number Enter n");
+        String choice = UserInput.get();
+        if (choice.equals("t")) {
+            changeBike(idBike, "title");
+        }
+        if (choice.equals("p")) {
+            changeBike(idBike, "price");
+        }
+        if (choice.equals("n")) {
+            changeBike(idBike, "number");
+        }
+    }
+
+    private void changeBike(int idBike, String value) {
+        Bike bike = bikes.get(idBike);
+        if (value.equals("title")) {
+            System.out.println("Old title : " + bike.getTitle() + "\nEnter new title :");
+            bike.setTitle(UserInput.get());
+        }
+        if (value.equals("price")) {
+            System.out.println("Old price : " + bike.getPrice() + "\nEnter new price :");
+            bike.setPrice(Integer.parseInt(UserInput.get()));
+        }
+        if (value.equals("number")) {
+            System.out.println("Old number : " + bike.getNumber() + "\nEnter new number :");
+            bike.setNumber(Integer.parseInt(UserInput.get()));
+        }
+        bikes.set(idBike, bike);
+    }
+
+    private void addNewBike() {
+        int addNew = bikes.size();
+        bikes.add(new Bike("newBike", 0, 0));
+        changeBike(addNew, "title");
+        changeBike(addNew, "price");
+        changeBike(addNew, "number");
+        System.out.println("New Bike Created");
+    }
+
+    private void deleteOldBike(int id) {
+        int bikeSize = bikes.size();
+        if (id < bikeSize && bikeSize != 0) {
+            System.out.println("Old Bike : " + bikes.get(id).getTextBikePresentation());
+            System.out.println("Delet : Enter y or n");
+            String choice = UserInput.get();
+            if (choice.equals("y")) {
+                bikes.remove(id);
+            }
+            if (choice.equals("n")) {
+                return;
+            }
+        }
+        if (bikeSize == 0) {
+            System.out.println("Empty base");
+        }
+    }
+
+    private void getBikesFromTxt() {
```

Comment:
Move this method to a separate class "BikesDao" and rename as "findAll"

Reply:
ok